packages/api/src/controllers/stream.ts

```diff
@@ -1,15 +1,12 @@
-import { Router, Request } from "express";
+import { Request, Router } from "express";
+import _ from "lodash";
```

Comment:
Probably imported by mistake?

Reply:
Ah, I see it was elsewhere

Reply:
No it's used in this file
https://github.com/livepeer/livepeer-com/blob/2a52015e9c4a0a41d94b7fe5ba5996af9447b097/packages/api/src/controllers/stream.ts